gordo_components/cli/cli.py

```diff
@@ -169,7 +178,10 @@ def build(
                 Example::
 
                     {"cv_mode": "cross_val_only"}
+    enable_remote_logging: bool
+        Flag to enable remote logging of model building results.
```

Comment:
Maybe add "to Azure ML services" and that otherwise a local mlflow folder will be created?